lib/content-services/src/lib/aspect-list/aspect-list.component.html

```diff
@@ -1,18 +1,18 @@
 <div id="aspect-list-container" class="adf-aspect-list-container" *ngIf="aspects$ | async as aspects; else loading">
     <mat-accordion class="adf-accordion-aspect-list">
-        <mat-expansion-panel *ngFor="let aspect of aspects; let colIndex = index" [id]="'aspect-list-'+aspect?.entry?.title">
-            <mat-expansion-panel-header [id]="'aspect-list-'+aspect?.entry?.title+'header'">
+        <mat-expansion-panel *ngFor="let aspect of aspects; let colIndex = index" [id]="'aspect-list-'+(aspect?.entry?.title ? aspect?.entry?.title : aspect?.entry?.id.replace(':', '-'))">
+            <mat-expansion-panel-header [id]="'aspect-list-'+(aspect?.entry?.title ? aspect?.entry?.title : aspect?.entry?.id.replace(':', '-'))+'header'">
```

Comment:
can we have a method getId() that will contain this stuff?

Reply:
I agree with @arditdomi better to use a method plus do we need all these question marks? is it a bit overdefensive?